storage/service_account/hmac/activate.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hmac
+
+// [START storage_activate_hmac_key]
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+
+	"cloud.google.com/go/storage"
+)
+
+// Activate the HMAC key with the given access ID.
+func activateHMACKey(w io.Writer, accessID string, projectID string) (*storage.HMACKey, error) {
+	ctx := context.Background()
+
+	// Initialize client.
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+		return nil, err
+	}
+	defer client.Close() // Closing the client safely cleans up background resources.
+
+	key_handle := client.HMACKeyHandle(projectID, accessID)
```

Comment:
rename `key_handle` => `handle`

Reply:
Ah, I changed this to keyHandle in the test because of the preference for camel case but missed it here. I guess handle would be even better in all places (for the sake of brevity)?